upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1384,5 +1385,35 @@ func (n *nodeUpConfigBuilder) BuildConfig(ig *kops.InstanceGroup, apiserverAddit
 	config.Channels = n.channels
 	config.EtcdManifests = n.etcdManifests[role]
 
+	buildContainerdConfig(config, cluster)
```

Comment:
Should preserve the style.
```suggestion
	config.ContainerdConfig = buildContainerdConfig(cluster)
```

Reply:
You mean moving this logic to the apis packages? I don't think this logic really has much to do with the API itself.
Given that we move stuff from the Cluster and IG resources to Nodeup.Config, we probably need more dedicated builders at some point similar to how we populate IG and Cluster.

Reply:
Meant that you pass the `config` arg just to set `config.ContainerdConfig` at the end. So, instead of passing it to set it, just return a string from the builder.

Reply:
yeah I realised I didn't read that right. sorry about that. Should be better now.